modules/Story/Story.tsx

```diff
@@ -40,35 +40,34 @@ function Story({ story }: Props) {
         <Layout>
             <StorySeo story={story} />
             <article className={styles.story}>
-                {headerImage && (
-                    <Image
-                        alt=""
-                        className={classNames({
-                            [styles.mainImage]: !IS_FANCY_IMAGE_ENABLED,
-                            [styles.fullWidthImage]: IS_FANCY_IMAGE_ENABLED,
-                        })}
-                        objectFit="cover"
-                        layout="fill"
-                        imageDetails={headerImage}
-                    />
-                )}
                 <div
                     className={classNames(styles.container, {
                         [styles.withImage]: hasHeaderImage && !IS_FANCY_IMAGE_ENABLED,
                         [styles.withFullWidthImage]: hasHeaderImage && IS_FANCY_IMAGE_ENABLED,
                     })}
                 >
-                    <StoryStickyBar story={story} />
-                    {isEmbargoStory(story) && <Embargo story={story} />}
-                    <div className={styles.meta}>
-                        {showDate && <StoryPublicationDate story={story} />}
-                        {showDate && hasCategories && <> &middot; </>}
-                        {hasCategories && (
-                            <CategoriesList categories={categories} showAllCategories />
-                        )}
-                    </div>
+                    {hasCategories && <CategoriesList categories={categories} showAllCategories />}
                     <h1 className={styles.title}>{title}</h1>
                     <p className={styles.subtitle}>{subtitle}</p>
+                    {showDate && (
+                        <p className={styles.date}>
+                            <StoryPublicationDate story={story} />
+                        </p>
+                    )}
+                    <StoryLinks url={links.short || links.newsroom_view} />
```

Comment:
❌ 
The popup shown after clicking the Copy URL button looks weird:

<img width="238" alt="Screen Shot 2022-07-20 at 12 49 34" src="https://user-images.githubusercontent.com/1641218/179941689-05f95d97-ac37-481e-98db-6a4dc31a68ec.png">

Reply:
Updated the design